### ISSUE TITLE
Deploy to Heroku Not Working?

### ISSUE THREAD

**Issue by adias9** [techx/quill#109]
I need some help.... It is slightly dire.

It seems to work, when I run it locally, but using the deploy now button and trying to push my version to heroku the deployment does not work.  Is there some other sort of adjustments I need to make for it to work for heroku? This is my first time working with angular.

**Comment by horeilly1101** [comment]
Were you able to find a solution? I’m having the same problem

**Comment by adias9** [comment]
Yes I had to reconfigure some stuff though and redo a lot. It was several months ago, so I don’t remember what I did exactly. But check out the configuration variables

**Comment by horeilly1101** [comment]
Thanks for your response! Would you be comfortable sharing your code with me? I still haven't been able to figure it out

**Comment by mattorgill** [comment]
I just tried to use the deploy to Heroku button without trying to push any of my own changes and it wouldn't work. Any solutions?

**Comment by horeilly1101** [comment]
I wasn't able to get the current version (or at least the current version on April 14) to deploy to heroku, so I just deployed off an earlier commit. Clicking the button here should work: https://github.com/techx/quill/tree/226cf7365d7123dd73213c51491e30068e6b48fd

**Comment by mattorgill** [comment]
Gotcha, thanks!
